---
description: "詳細情報: <error>: '<constructorname1>' が '<constructorname2>' を呼び出しています"
title: "<error>: '<constructorname1>' が '<constructorname2>' を呼び出しています"
ms.date: 07/20/2015
f1_keywords:
- vbc30297
- bc30297
helpviewer_keywords:
- BC30297
ms.assetid: dfca67d7-f4d7-4451-a937-68f22b8527d5
ms.openlocfilehash: 3e965399e384761e81f3ab894ad0c7668d745783
ms.sourcegitcommit: 10e719780594efc781b15295e499c66f316068b8
ms.translationtype: HT
ms.contentlocale: ja-JP
ms.lasthandoff: 02/14/2021
ms.locfileid: "100425374"
---
# <a name="error-constructorname1-calls-constructorname2"></a>\<error>: '\<constructorname1>' が '\<constructorname2>' を呼び出しています

コンストラクターの呼び出しが循環しています。 コンストラクターでは、 `Me.New()` または `MyClass.New()`が呼び出されます。 異なる引数リストを持つ、オーバーロードされたコンストラクターを呼び出そうとしたことが、原因の 1 つとして考えられます。  
  
 **エラー ID:** BC30297  
  
## <a name="to-correct-this-error"></a>このエラーを解決するには  
  
- 異なる引数リストを使用して、オーバーロードされたコンストラクターを呼び出します。  
  
- アクセス可能なオーバーロードされたコンストラクターがない場合は、 `Me.New()` および `MyClass.New()`の呼び出しを削除します。  
  
## <a name="see-also"></a>関連項目

- [コンストラクター](../programming-guide/concepts/object-oriented-programming.md#constructors)
